src/Aspire.Cli/Aspire.Cli.csproj

```diff
@@ -26,7 +26,10 @@
     <RollForward>Major</RollForward>
     <PackageTags>aspire cli</PackageTags>
     <Description>Command line tool for Aspire developers.</Description>
+    <!-- For 9.5 we want to keep the tool platform-independent, framework-dependent.
```

Comment:
Do we want to keep bring this change over to main?

Reply:
It looks like @baronfel was making the same change in main in https://github.com/dotnet/aspire/pull/11609.

I'm not sure why we'd want "For 9.5" in the comment though. Shouldn't it be for all versions?

Reply:
IIRC the thinking was in a subsequent version we'd try to tackle the 'cross-compatible RID-specific tool pattern', which would require more changes to this file so the 9.5 context was to time-box the rationale.